themes/finna2/templates/checkouts/history.phtml

```diff
@@ -90,7 +90,7 @@
                               <button class="btn btn-primary dropdown-toggle download" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                                 <?=$this->transEsc('loan_history_download')?> <span class="caret"></span>
                               </button>
-                              <ul class="dropdown-menu dropdown-menu-right download">
+                              <ul class="dropdown-menu dropdown-menu-<?php if ($purgeSelectedAllowed): ?>right<?php else: ?>left<?php endif;?> download">
```

Comment:
Minkähän takia tuossa edes on tuo right? Jos vain ottaisi kokonaan pois?

Reply:
Tässä taisi olla että default (siis ilman) on että se kääntyy oikealle päin, mutta se sitten taas menee lopulta yli oikeasta reunasta mobiilissa, jos on ne viereiset napit/valinnat. Ja jos laitan säännön, että mobiilissa kääntyy vasemmalle niin ilman viereisiä nappeja se menee sieltä puolelta yli.

Reply:
Ok.